### ISSUE TITLE
Token has html content suddenly

### ISSUE THREAD

**Issue by TomDeSmet** [verbb/formie#284]
**Description**
After updating from 1.3.4 to 1.3.7, adding the e-mail token field adds html content suddenly which fails to pass the RFC test for sending notifications.

```
[error][formie] Notification email parse error for “To: {field.email}”. Template error: “Address in mailbox given [<p><strong>Email</strong><br>xxx@the-craft.be</p>] does not comply with RFC 2822, 3.6.2.” /data/sites/web/orielwindcom/subsites/releases/20210108134756/vendor/swiftmailer/swiftmailer/lib/classes/Swift/Mime/Headers/MailboxHeader.php:355
```

Rolling back to 1.3.4 fixes this issue.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.7",
  "...": "..."
}
```

Then run `composer update`. You may also need to remove the variable tag for the field and add it again. We've recently re-worked how the content for these variables are parsed/stored, as you've noticed by downgrading.

**Comment by engram-design** [comment]
Fixed in [1.3.8](https://github.com/verbb/formie/releases/tag/1.3.8).
